src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -407,7 +410,16 @@ const FileField = props => {
 
   const uploadText = content[files.length > 0 ? 'uploadAnother' : 'upload'];
 
-  return (
+  return uiOptions.uswds && !formContext.reviewMode ? (
+    // Use v3 multi-file upload:
+    <VaFileInputMultiple
+      button-text="Upload your document"
+      onVaMultipleChange={e => onAddFile({ target: e.detail })}
```

Comment:
I think `onAddFile` may need to be adjusted slightly for v3 too - [example here](https://github.com/department-of-veterans-affairs/vets-website/blob/9af621cdbd8e2acf28fa2fc891abdca891cf5838/src/platform/forms-system/src/js/fields/FileField.jsx#L237) (possibly not complete)